### PR TITLE
feat(pronouns.cc): init

### PR DESCRIPTION
**Pull request by comfysage**
## 🎉 Theme for [pronouns.cc](https://pronouns.cc) 🎉

preview-image

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/scripts/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - [x] `catppuccin.user.css` - all the CSS for the userstyle, based on the
        template.
  - [x] `preview.webp` - composite image of all four individual flavor screenshots (taken with the default accent color of mauve) stitched together, generated via [Catwalk](https://github.com/catppuccin/catwalk).

### CODE REVIEW

**Review by isabelroses — CHANGES_REQUESTED** [review]
you might consider theming `--bs-nav-link-color` as subtext0 so theres more of a diffrence with active links.

There's too much contrast here:
{12E81ABE-439D-42C0-85ED-816C158469C9}
This can be found on a users page.

It also strips the borderes
{527E2B09-EC13-4225-9651-B954D9F87CD2}

**Review by isabelroses — CHANGES_REQUESTED** [review]
We seem to now have other broken buttons
{7BB6BEEA-D260-45E5-98F0-E56DCCBBF144}
image

**Review by isabelroses — APPROVED** [review]

**Review by isabelroses — DISMISSED** [review]

**Review by uncenter — CHANGES_REQUESTED** [review]
The border on this tooltip in the adding pronouns UI appears unthemed:

CleanShot 2024-11-15 at 19 38 17

Border and selected item in this dropdown UI on the edit profile page (Preferences section) are unthemed:

CleanShot 2024-11-15 at 19 39 08

Borders for the cards on https://pronouns.cc/settings/auth unthemed.

Menu toggle in the top right is unthemed:

CleanShot 2024-11-15 at 19 41 39

Close icons in modals like this log out one are unthemed:

CleanShot 2024-11-15 at 19 41 58

**Review by isabelroses — APPROVED** [review]

**Review by sgoudham — DISMISSED** [review]

File: styles/pronouns.cc/catppuccin.user.css [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+      @svg: escape('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" fill="@{surface1}"><path d="M.293.293a1 1 0 0 1 1.414 0L8 6.586 14.293.293a1 1 0 1 1 1.414 1.414L9.414 8l6.293 6.293a1 1 0 0 1-1.414 1.414L8 9.414l-6.293 6.293a1 1 0 0 1-1.414-1.414L6.586 8 .293 1.707a1 1 0 0 1 0-1.414"/></svg>');
+      --bs-btn-close-bg: url("data:image/svg+xml,@{svg}");
+      --bs-btn-close-opacity: 1.0;
+      --bs-btn-close-hover-opacity: .75;
+      --bs-btn-close-focus-shadow: 0 0 0 .25rem rgba(13, 110, 253, .25);
+      --bs-btn-close-focus-opacity: 1;
+      --bs-btn-close-disabled-opacity: .25;
+      --bs-btn-close-white-filter: none;
+      color: var(--bs-btn-close-color);
+      opacity: var(--bs-btn-close-opacity);
```
Comment (uncenter) [inline styles/pronouns.cc/catppuccin.user.css:245]: Do we need to set these?
```suggestion
```

**Review by uncenter — CHANGES_REQUESTED** [review]

File: styles/pronouns.cc/catppuccin.user.css (hunk 1 above)
Comment (comfysage) [reply styles/pronouns.cc/catppuccin.user.css:245]: we actually need the first line but the rest can go.

**Review by uncenter — APPROVED** [review]

**Pull request merged**